client/settings/settings-manager/index.js

```diff
@@ -83,23 +83,32 @@ const SettingsManager = ( {
 	);
 
 	return (
-		<div className="settings-manager">
-			<SettingsSection Description={ PaymentMethodsDescription }>
-				<PaymentMethods
-					enabledMethodIds={ enabledPaymentMethodIds }
-					onEnabledMethodIdsChange={ setEnabledPaymentMethodIds }
-				/>
-			</SettingsSection>
-			<SettingsSection Description={ DigitalWalletsDescription }>
-				<DigitalWallets />
-			</SettingsSection>
-			<SettingsSection Description={ GeneralSettingsDescription }>
-				<GeneralSettings accountLink={ accountStatus.accountLink } />
-			</SettingsSection>
-			<SettingsSection>
-				<TestModeSettings />
-			</SettingsSection>
-		</div>
+		<>
+			<Card size="large" className="settings-manager__banner">
```

Comment:
What do you think about moving this to a separate component (with its own styles) so that it can be reused in the "1-click checkout" page? https://d.pr/i/QRNxqD